src/components/InLineNotification.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { fireEvent, render } from '@testing-library/react-native'
+import * as React from 'react'
+import InLineNotification, { Severity } from 'src/components/InLineNotification'
+
+describe(InLineNotification, () => {
+  it('does not render CTA when onPress function is not provided', async () => {
+    const { findByText } = render(
```

Comment:
you should be able to use `getByText` here - this component isn't doing anything async before rendering. you'll get better errors with `getByText` too.